blackjax/mcmc/random_walk.py

```diff
@@ -270,22 +270,27 @@ def kernel(
         state: RWState,
         logdensity_fn: Callable,
         proposal_distribution: Callable,
-    ) -> tuple[RWState, RWInfo]:
+        proposal_logdensity_fn: Optional[Callable] = None,
+    ) -> Tuple[RWState, RWInfo]:
         """
-
         Parameters
         ----------
         proposal_distribution
             A function that, given a PRNGKey, is able to produce a sample in the same
             domain of the target distribution.
+        proposal_logdensity_fn:
+            For non-symmetric proposals, a function that returns the log-density
+            to obtain a given proposal knowing the current state. If it is not
+            provided we assume the proposal is symmetric.
         """
 
         def proposal_generator(rng_key: PRNGKey, position: ArrayTree):
-            del position
```

Comment:
reverse.

Reply:
done, just out of curiosity why is that del needed?

Reply:
It's not needed, mostly it is styling to signify the input arg is not needed.